test/jasmine/tests/plot_interact_test.js

```diff
@@ -142,6 +150,119 @@ describe('Test plot structure', function() {
             });
         });
 
+        describe('scatter drag', function() {
+
+            var mock = require('@mocks/scatter_simple.json'),
```

Comment:
no need to add another mock, there are plenty of cartesian mocks to choose from in `@mocks`